File: Java/Medium/WrongAnswer/sort-the-jumbled-numbers_4.java
class Solution {
    public int[] sortJumbled(int[] mapping, int[] nums) {
        
        int[] sorted = new int[nums.length];
        for (int i = 0; i < nums.length; i++) {
            int num = nums[i];
            int newNum = 0;
            while (num > 0) {
                newNum = newNum * 10 + mapping[num % 10];
                num /= 10;
            }
            sorted[i] = newNum;
        }
        Arrays.sort(sorted);
        return sorted;
    }
}

File: C++/Medium/WrongAnswer/sort-the-jumbled-numbers_4.cpp
class Solution {
public:
    vector<int> sortJumbled(vector<int>& mapping, vector<int>& nums) {
        
        vector<int> ans;
        for (int i = 0; i < nums.size(); i++) {
            int num = nums[i];
            int newNum = 0;
            while (num) {
                newNum = newNum * 10 + mapping[num % 10];
                num /= 10;
            }
            ans.push_back(newNum);
        }
        sort(ans.begin(), ans.end());
        return ans;
    }
};